src/yunohost/regenconf.py

```diff
@@ -43,14 +43,16 @@
 logger = log.getActionLogger('yunohost.regenconf')
 
 
+# FIXME : those ain't just services anymore ... what are we supposed to do with this ...
```

Comment:
Can't we migrate those?

Reply:
You mean the existing logs ? Meh I guess it's not so much of a big deal to not migrate them, usually you only care about the recent logs

I was more thinking about if we can change this `'service'` key in the `is_unit_operation` and I guess we can indeed replace it to something like `configuration` ?

Reply:
or "regen-conf"?